src/main/res/values-te/strings.xml

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name = "alert_delete_confirm"> మీకు ఖచ్చితంగా తెలుసా? </ string>
```

Comment:
Please remove extra space before and after each entry. Use refactoring in Android studio